src/app/modules/forms/ix-forms/components/ix-explorer/explorer-create-dataset/explorer-create-dataset.component.ts

```diff
@@ -0,0 +1,78 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  input,
+  computed,
+} from '@angular/core';
+import { toSignal } from '@angular/core/rxjs-interop';
+import { NgControl } from '@angular/forms';
+import { MatButton } from '@angular/material/button';
+import { MatDialog } from '@angular/material/dialog';
+import { UntilDestroy, untilDestroyed } from '@ngneat/until-destroy';
+import { TranslateModule } from '@ngx-translate/core';
+import { filter } from 'rxjs';
+import { RequiresRolesDirective } from 'app/directives/requires-roles/requires-roles.directive';
+import { Role } from 'app/enums/role.enum';
+import { Dataset, DatasetCreate } from 'app/interfaces/dataset.interface';
+import { CreateDatasetDialog } from 'app/modules/forms/ix-forms/components/ix-explorer/create-dataset-dialog/create-dataset-dialog.component';
+import { IxExplorerComponent } from 'app/modules/forms/ix-forms/components/ix-explorer/ix-explorer.component';
+import { IxIconComponent } from 'app/modules/ix-icon/ix-icon.component';
+import { TestDirective } from 'app/modules/test-id/test.directive';
+
+@UntilDestroy()
+@Component({
+  selector: 'ix-explorer-create-dataset',
+  templateUrl: './explorer-create-dataset.component.html',
+  styleUrls: ['./explorer-create-dataset.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  standalone: true,
+  imports: [
+    MatButton,
+    IxIconComponent,
+    TranslateModule,
+    RequiresRolesDirective,
+    TestDirective,
+  ],
+})
+export class ExplorerCreateDatasetComponent {
+  readonly datasetProps = input<Omit<DatasetCreate, 'name'>>({});
```

Comment:
This name could be better. Also, since we override the name property in dialog submission when creating the dataset anyway, omitting name here seems unnecessary detail.

Reply:
Cannot change the type, because `name` is required in `DatasetCreate` and I would have to cast here.